src/test/resources/cassandra-reaper-cassandra-at.yaml

```diff
@@ -8,6 +8,7 @@ repairRunThreadCount: 15
 hangingRepairTimeoutMins: 1
 storageType: cassandra
 incrementalRepair: false
+allowUnreachableNodes: true
```

Comment:
Integration tests wouldn't work without turning this on.  Still not sure if i broke something, if my environment changed, or if I fixed something…?